spyderlib/config.py

```diff
@@ -169,6 +169,7 @@ def is_ubuntu():
               'animated_docks': True,
               'prompt_on_exit': False,
               'panes_locked': True,
+              'interface_language': load_lang_conf(),

```

Comment:
@ccordoba12 

If I don't add that method to the config, Spyder will not be able to load the preferences after a reset, cause there will be no `interface_language` setting in the .ini file.

Suggestions?